src/docs/patterns/user-auth.mdx

```diff
@@ -0,0 +1,324 @@
+import VerifyEmail from '../../app/assets/design-patterns/user-authentication/verify-email.png';
+import CreatePassword from '../../app/assets/design-patterns/user-authentication/create-password.png';
+import AccountDetails from '../../app/assets/design-patterns/user-authentication/account-details.png';
+import WelcomeOnboard from '../../app/assets/design-patterns/user-authentication/welcome-onboard.png';
+import RegistrationDesktop from '../../app/assets/design-patterns/user-authentication/registration-desktop.png';
+
+import EnterOrganizationCode from '../../app/assets/design-patterns/user-authentication/enter-organization-code.png';
+import EnterOrganizationPasscode from '../../app/assets/design-patterns/user-authentication/enter-organization-passcode.png';
+import EnterOrganizationConfirm from '../../app/assets/design-patterns/user-authentication/enter-organization-confirm.png';
+import EnterOrganizationWelcome from '../../app/assets/design-patterns/user-authentication/enter-organization-welcome.png';
+import JoinAnOrganizationDesktop from '../../app/assets/design-patterns/user-authentication/join-an-organization-desktop.png';
+
+import Login from '../../app/assets/design-patterns/user-authentication/login.png';
+import LoginDesktop from '../../app/assets/design-patterns/user-authentication/login-desktop.png';
+
+import ForgotPasswordEnterEmail from '../../app/assets/design-patterns/user-authentication/forgot-password-email.png';
+import ForgotPasswordConfirmSent from '../../app/assets/design-patterns/user-authentication/forgot-password-confirm.png';
+import ForgotPasswordNewPassword from '../../app/assets/design-patterns/user-authentication/forgot-password-password.png';
+import ForgotPasswordWelcome from '../../app/assets/design-patterns/user-authentication/forgot-password-welcome.png';
+import ChangePassword from '../../app/assets/design-patterns/user-authentication/change-password.png';
+import ChangePasswordWelcome from '../../app/assets/design-patterns/user-authentication/change-password-confirm.png';
+import ResetPasswordDesktop from '../../app/assets/design-patterns/user-authentication/reset-password-desktop.png';
+import ChangePasswordDesktop from '../../app/assets/design-patterns/user-authentication/change-password-desktop.png';
+
+<!-- Behaviors -->
+
+import EmailCommunicationInvitation from '../../app/assets/design-patterns/user-authentication/email-communication-invitation.png';
+import EmailCommunicationVerification from '../../app/assets/design-patterns/user-authentication/email-communication-verification.png';
+import ExpiredCode from '../../app/assets/design-patterns/user-authentication/expired.png';
+import ErrorStates from '../../app/assets/design-patterns/user-authentication/error-states.png';
+
+<!-- KEYWORDS: design pattern login registration register password forget forgot account authentication auth join -->
+
+# User Authentication & Registration
```

Comment:
Should we just call this thing "Login & Registration"? Will that fit on one line in the drawer?

Reply:
Maybe not. "User Authentication" in the sidebar doesn't seem like it covers everything that is in here (registration). It's probably fine though.